packages/paypal-commerce-integration/src/paypal-commerce-credit/paypal-commerce-credit-customer-strategy.ts

```diff
@@ -234,7 +234,11 @@ export default class PayPalCommerceCreditCustomerStrategy implements CustomerStr
             await this.paymentIntegrationService.selectShippingOption(shippingOption.id);
             await this.paypalCommerceIntegrationService.updateOrder();
         } catch (error) {
-            throw new Error(error);
+            if (typeof error === 'string') {
+                throw new Error(error);
+            }
+
+            throw error;
```

Comment:
Lets change it with `this.handleError(error)`

Reply:
As this type check is definitely working, I think we don't need to combine `throw new Error(error)` into the exsiting `private handleError(error: unknown) {` logic.

Reply:
I think we have to update catch implementation with this.handleError(error) because it gives an ability to handle error through `onError` option provided from UI

Reply:
Thanks for the feedback.
I misunderstood before. Thought you were suggestting adding `throw new Error(error);` to the `this.handleError`.
Could you please check if I got your idea right in this commit, https://github.com/bigcommerce/checkout-sdk-js/pull/2650/commits/b99ba8850bf80188170807c1a9d6565322666dfb ?